drivers/rtc/rtc_ll_stm32.c

```diff
@@ -79,22 +79,10 @@ struct rtc_stm32_data {
 
 static int rtc_stm32_enter_initialization_mode(bool kernel_available)
```

Comment:
You can remove the kernel_available argument wich is now useless.

Reply:
I guess the whole function is useless.

Reply:
I removed that function and replaced with LL function.

Reply:
Well then if you removed it you can also remove the rtc_stm32_leave_initialization_mode function.

Reply:
Yes, that makes sense.